app/assets/javascripts/components/DemographicInfoForm.coffee

```diff
@@ -0,0 +1,155 @@
+{ DOM: { div, input, option, span, small, ul, li, p, label, button, form, i } } = React
+{ reduxForm } = ReduxForm
+
+demographicInfoForm = React.createClass
+  displayName: 'DemographicInfoForm'
+
+  questions: [
+    (
+      id: 'gender'
+      intro: 'Voor onderzoek vragen we u om een paar vragen te beantwoorden over uw achtergrond. Door op verzenden te klikken gaat u akkoord met de voorwaarden.'
```

Comment:
Welke voorwaarden? :)
De tekst op de knop is afronden, niet verzenden.
Is het niet een beetje flauw dat we **vragen** om een paar vragen te beantwoorden, maar dat je dat wel verplicht moet doen?